README.md

```diff
@@ -42,18 +55,9 @@ the form:
 
     PANOPTICON_API_CREDENTIALS = { bearer_token: 'MY_BEARER_TOKEN' }
 
-
-## Middleware for request tracing
-
-We set a unique header at the cache level called `GOVUK-Request-Id`. In order
-to serve a user's request, if apps make API requests they should pass on this
-header, so that we can trace a request across the entire GOV.UK stack.
-
-`GdsApi::GovukRequestIdSniffer` middleware takes care of this. This gem contains
-a railtie that configures this middleware for Rails apps without extra effort.
-Other Rack-based apps should opt-in by adding this line to your `config.ru`:
-
-```use GdsApi::GovukRequestIdSniffer```
+In addition, the adapters can pass on a header containing the authenticated user
```

Comment:
I've tweaked my comment above slightly. The real distinction is that the `bearer_token` is used for gds-sso-based **authentication**, and the authenticated user header is used to identify an authenticated user so that the app is able to perform additional **authorisation** if required.